### PR TITLE
Feature/new landing

### PR DESCRIPTION
**Pull request by jmzwar**
New marketing website landing page.

### CODE REVIEW

File: src/sections/home/hero.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { Button, Flex, Text } from '@chakra-ui/react';
+import SNXMainLooperPlaceholder from 'src/svg/SNXMainLooperPlaceholder';
+
+export const Hero = () => {
+	let onClick = () => {};
+
+	if (typeof window !== 'undefined') {
```
Comment (bachstatter) [inline src/sections/home/hero.tsx:7]: Slightly nicer to just put the `typeof window !== 'undefined'` inside onClick?

File: src/sections/home/integrators.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { Flex } from '@chakra-ui/react';
+import { IntegratorCard } from 'src/components';
+import { integrators } from 'src/utils/integrators';
+
+interface IntegratorsProps {
+	sortList: string[];
+}
+
+export const Integrators = ({ sortList }: IntegratorsProps) => {
+	// TOTAL HACK
```
Comment (bachstatter) [inline src/sections/home/integrators.tsx:10]: What does Total Hack mean?

File: src/sections/home/perps.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { Box, Button, Flex, Text } from '@chakra-ui/react';
+import { GradientText } from 'src/components';
+
+export const Perps = () => {
+	let onClick = () => {};
```
Comment (bachstatter) [inline src/sections/home/perps.tsx:5]: Same thing here, I think conditionally mutating an onClick variable feels a little off

File: src/typings/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,101 @@
+export type StakedSNXResponse = {
```
Comment (bachstatter) [inline src/typings/index.d.ts:1]: d.ts file can behave a little wierd. I would only use that if you're adding type to an untyped package. Change this to index.ts or types.ts

File: src/typings/index.d.ts [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+	systemStakingPercent: number;
+	timestamp: number;
+	stakedSnx: {
+		ethereum: number;
+		optimism: number;
+	};
+};
+
+export type VolumeData = {
```
Comment (bachstatter) [inline src/typings/index.d.ts:10]: Another thought, who owns all of the dune dashboards we're grabbing data from? If this they change a query and click save they will break this?

Might be worth thinking of some error handling and maybe runtime validation with zod

**Review by bachstatter — APPROVED** [review]
Left a few comments, nothing blocking. Clicked around a little on the deployed preview version too

File: src/typings/index.d.ts (hunk 5 above)
Comment (jmzwar) [reply src/typings/index.d.ts:10]: Yeah I've taken the easy approach on this - just not showing data if there is an issue, but good thoughts if we are investing more on the data side of things

File: src/sections/home/perps.tsx (hunk 3 above)
Comment (jmzwar) [reply src/sections/home/perps.tsx:5]: yup updating

File: src/sections/home/hero.tsx (hunk 1 above)
Comment (jmzwar) [reply src/sections/home/hero.tsx:7]: +1

File: src/sections/home/integrators.tsx (hunk 2 above)
Comment (jmzwar) [reply src/sections/home/integrators.tsx]: Updated the comment

File: pages/index.tsx [hunk 6]
```diff
@@ -1,90 +1,233 @@
...
+			{ data: volumeData },
+			{ data: activeStakersData },
+			{ data: integratorsData },
+			{ data: openInterestData },
+			{ data: tradingFeesData },
+			{ data: graphqlResponse },
+		] = await Promise.all([
 			axios.get<StakedSNXResponse>(STAKED_SNX_DATA_URL),
 			getSnxPrice(),
+			axios.get<VolumeResponse>(VOLUME_DATA_URL, {
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply pages/index.tsx:139]: Yeah we were using axios so just kept it.

File: src/typings/index.d.ts (hunk 4 above)
Comment (jmzwar) [reply src/typings/index.d.ts:1]: Sure

File: src/svg/StakingLooper.tsx [hunk 7]
```diff
@@ -0,0 +1,1620 @@
+import { Icon, IconProps } from '@chakra-ui/react';
+
+export const StakingLooper = ({
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply src/svg/StakingLooper.tsx:3]: yeah boi

**Pull request merged**